src/main/resources/assets/computercraft/lang/de_de.lang

```diff
@@ -1,3 +1,5 @@
+itemGroup.computercraft=CC: Tweaked
```

Comment:
This is not necessary - the mod name should never be translated, and so should only exist in `en_us.lang`.

Reply:
I think, it is better to put it in every language files, because some tools for search for missing entry's may find this and add it automatically or throw a warning for a missed entry.